cmd/exec/exec.go

```diff
@@ -42,15 +43,24 @@ func ExecutePlan(commands ...string) {
 }
 
 func executeAndStopIfFailed(command *models.CommandContext, executeStart time.Time) {
-	command.PrintPhaseBanner()
-	err := command.CommandSession.Run()
-	if err != nil {
+	stdout, err := command.CommandSession.CombinedOutput()
```

Comment:
With this - will you get realtime output when you dont specify flag.

Example - if you run gradle task which takes 2 min to finish - Will this show realtime output logs of gradle or entire log will be visible after 2 min?

We want the first one - see realtime output

Reply:
will change implementation.